kvm/v12.2.0-2020092800/release.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v12.2.0-2020092800
+spec:
+  apps:
+    - name: cert-exporter
+      version: 1.2.3
+    - name: chart-operator
+      version: 2.3.2
+    - componentVersion: 1.6.5
+      name: coredns
+      version: 1.2.0
+    - componentVersion: 1.9.7
+      name: kube-state-metrics
+      version: 1.1.1
+    - componentVersion: 0.3.6
+      name: metrics-server
+      version: 1.1.1
+    - name: net-exporter
+      version: 1.9.0
+    - componentVersion: 1.0.1
+      name: node-exporter
+      version: 1.3.0
+  components:
+    - name: app-operator
+      version: 2.3.1
```

```suggestion
      version: 2.3.2
```

Can we use the latest release?